src/migrations/20220714052431-create-user.js

```diff
@@ -10,18 +10,50 @@ module.exports = {
         type: Sequelize.INTEGER,
       },
       firstName: {
+<<<<<<< HEAD
```

Comment:
```suggestion
<<<<<<< HEAD
```

Hey Kim,

Looks good! Only issue I can see is just some leftover merge text, can you just clean these up and push the commit through and I'll merge it in? 😄 Cheers